cfme/tests/configure/test_timeprofile.py

```diff
@@ -2,82 +2,113 @@
 import fauxfactory
 import pytest
 
-import cfme.configure.settings as st
 from cfme import test_requirements
-from cfme.configure.settings import TimeProfileAddFormView
+from cfme.configure.settings import TimeProfileAddView
 from cfme.utils.update import update
 
+
 pytestmark = [pytest.mark.tier(3),
               test_requirements.settings]
 
 
-def new_timeprofile():
-    return st.Timeprofile(description='time_profile' + fauxfactory.gen_alphanumeric(),
-                          scope='Current User',
-                          days=True,
-                          hours=True,
-                          timezone="(GMT-10:00) Hawaii")
-
-
 @pytest.mark.sauce
-def test_timeprofile_crud():
-    timeprofile = new_timeprofile()
-    timeprofile.create()
-    with update(timeprofile):
-        timeprofile.scope = 'All Users'
-    copied_timeprofile = timeprofile.copy()
-    copied_timeprofile.delete()
-    timeprofile.delete()
+def test_time_profile_crud(appliance):
+    """
+        This test case performs the CRUD operation.
+    """
+    collection = appliance.collections.time_profiles
+    time_profile = collection.create(
+        description='time_profile {}'.format(fauxfactory.gen_alphanumeric()),
+        scope='Current User',
+        days=True, hours=True,
+        timezone="(GMT-10:00) Hawaii")
+    with update(time_profile):
+        time_profile.scope = 'All Users'
+    collection.delete(False, time_profile)
 
 
 @pytest.mark.sauce
-def test_timeprofile_name_max_character_validation():
-    tp = st.Timeprofile(
+def test_time_profile_name_max_character_validation(appliance):
+    """
+    This test case performs the check for max character validation.
+    """
+    collection = appliance.collections.time_profiles
+    time_profile = collection.create(
         description=fauxfactory.gen_alphanumeric(50),
         scope='Current User',
+        days=True, hours=True,
         timezone="(GMT-10:00) Hawaii")
-    tp.create()
-    tp.delete()
+    collection.delete(False, time_profile)
 
 
 @pytest.mark.sauce
-def test_days_required_error_validation(soft_assert, appliance):
-    tp = st.Timeprofile(
-        description='time_profile' + fauxfactory.gen_alphanumeric(),
+def test_days_required_error_validation(appliance, soft_assert):
+    """
+    This test case performs the error validation of days field.
+    """
+    collection = appliance.collections.time_profiles
+    collection.create(
+        description='time_profile {}'.format(fauxfactory.gen_alphanumeric()),
         scope='Current User',
         timezone="(GMT-10:00) Hawaii",
         days=False,
-        hours=True)
-    tp.create(cancel=True)
-    view = appliance.browser.create_view(TimeProfileAddFormView)
-    soft_assert(view.timeprofile_form.help_block.text == "At least one day needs to be selected")
-    soft_assert(view.timeprofile_form.save_button.disabled)
-    view.timeprofile_form.cancel_button.click()
+        hours=True,
+        cancel=True)
```

Comment:
**REQUIRED**
Here and below.
I believe it's not correct using of the collections. Here you create only an instance of `TimeProfile` class. In this test, I would instantiate a `TimeProfile` object, navigate to it and fill the view.

Reply:
There is no difference in using collection create method and the instantiate method as both the method will navigate to almost similar View (page) just the name of the timeprofile will be different as the edit page and create page is almost similar.

Reply:
That's what I'm talking about, `collection.create()` should always create an object which will exist in CFME, if you need just an instance use `instantiate()` method.